ci/doctests.sh

```diff
@@ -21,7 +21,7 @@ if [ "$DOCTEST" ]; then
 
     # DataFrame / Series docstrings
     pytest --doctest-modules -v pandas/core/frame.py \
-        -k"-assign -axes -combine -isin -itertuples -join -nlargest -nsmallest -nunique -pivot_table -quantile -query -reindex -reindex_axis -replace -round -set_index -stack -to_dict -to_records -to_stata -transform"
+        -k"-assign -axes -combine -isin -itertuples -join -nlargest -nsmallest -nunique -pivot_table -quantile -query -reindex -reindex_axis -replace -round -set_index -stack -to_dict -to_stata -transform"
```

Comment:
is this on-purpose?

Reply:
I was asked to make this change by @TomAugspurger in issue #18160. I believe the point is to check that the examples are correct.

Reply:
This is a whitelist of doctests that are allowed to fail till we fix them.

Reply:
Yep, going to open an issue about this